rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -77,29 +62,11 @@ object Reduce {
       * @param env  An environment marking the execution context.
       * @return  An optional continuation resulting from a match in the tuplespace.
       */
-    override def produce(chan: Quote, data: Seq[Par], persistent: Boolean)(
```

Comment:
`produce` and `consume` are now [extracted to `TuplespaceAlg`](https://github.com/goral09/rchain/blob/1ddaf64d9f7ae50c70ef4be7df617ac006d04e3f/rholang/src/main/scala/coop/rchain/rholang/interpreter/storage/TuplespaceAlg.scala#L21).